pyfstat/helper_functions.py

```diff
@@ -57,39 +56,7 @@ def set_up_command_line_arguments():
         action="store_true",
         help="Decrease output verbosity [logging.WARNING]",
     )
-    parser.add_argument(
-        "--no-interactive", help="Don't use interactive", action="store_true"
-    )
-    parser.add_argument(
-        "-c",
-        "--clean",
```

Comment:
looks sufficiently replaced by class options :+1:

Reply:
...or not: `AttributeError: 'GridGlitchSearch' object has no attribute 'clean'`

Reply:
Should be fixed by now.